pkg/report/writer.go

```diff
@@ -204,6 +204,9 @@ func NewTemplateWriter(output io.Writer, outputTemplate string) (*TemplateWriter
 		"escapeString": func(input string) string {
 			return html.EscapeString(input)
 		},
+		"replaceAll": func(input string, old string, new string) string {
```

Comment:
After merging https://github.com/aquasecurity/trivy/pull/832, you can use `replace`.
http://masterminds.github.io/sprig/strings.html

It is about to be merged. Just a moment, please.

Reply:
#832 got merged. Could you rewrite this function by `spring`?
http://masterminds.github.io/sprig/

Reply:
If 'sprig' functions are available to templates then this PR isn't required, and I can use 'replace' instead.

Reply:
> If 'sprig' functions are available to templates then this PR isn't required, and I can use 'replace' instead.

@sishbi are you able to provide an example how you made it in the end? Interested to use CSV output as well. thanks.

Reply:
@dcomas I have reopened the PR and used the 'replace' function from sprig for the csv.tpl